app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -334,6 +353,9 @@ import PreChatFormSettings from './PreChatForm/Settings';
 import WeeklyAvailability from './components/WeeklyAvailability';
 import GreetingsEditor from 'shared/components/GreetingsEditor';
 
+const shouldNoBeEmpty = (value = '') =>
```

Comment:
Move this logic to validation helpers

Reply:
@muhsin-k can you help me in fixing the code climate error.

Reply:
@iamsivin Add spec for `shouldNoBeEmpty`  methods